MSAL/src/native_auth/public/MSALNativeAuthUserAccountResult.swift

```diff
@@ -96,4 +100,38 @@ import Foundation
             }
         }
     }
+
```

Comment:
@borgesmb could you align with Android comment for getAccessToken and getAccessToken with scopes?